testplan/report/testing/schemas.py

```diff
@@ -71,44 +71,31 @@ class EntriesField(fields.Field):
     Handle encoding problems gracefully
     """
 
-    _BYTES_KEY = "_BYTES_KEY"
-
-    @staticmethod
-    def _binary_to_hex_list(binary_obj):
-        # make sure the hex repr is capitalized and leftpad'd with a zero
-        # because '0x0C' is better than '0xc'.
-        return [
-            "0x{}".format(hex(b)[2:].upper().zfill(2))
-            for b in bytearray(binary_obj)
-        ]
-
     @staticmethod
-    def _hex_list_to_binary(hex_list):
-        return bytes(bytearray([int(x, 16) for x in hex_list]))
+    def _json_serializable(v):
+        try:
+            json.dumps(v, ensure_ascii=True)
+        except (UnicodeDecodeError, TypeError):
+            return False
+        else:
+            return True
 
     def _serialize(self, value, attr, obj):
```

Comment:
Please add a comment about serialize behavior: return True - keep the node unchange, False - remove the mode, tuple - update the node data.